rusk-wallet/src/bin/command.rs

```diff
@@ -226,6 +227,28 @@ pub(crate) enum Command {
         gas_price: Lux,
     },
 
+    /// Claim accumulated rewards for a stake key
```

Comment:
```suggestion
    /// Claim accumulated stake rewards
```

The term "stake key" is no longer used, can you change the comment here? The terminology we use is "consensus key" and "owner key". "stake rewards" should be clear enough too and it simplifies the explanation of this command, in my opinion.